yfinance/scrapers/holders.py

```
@@ -96,8 +99,10 @@ def _fetch_and_parse(self):
             self._parse_insider_transactions(data.get("insiderTransactions", {}))
             self._parse_insider_holders(data.get("insiderHolders", {}))
             self._parse_net_share_purchase_activity(data.get("netSharePurchaseActivity", {}))
-        except (KeyError, IndexError):
-            raise YFDataException("Failed to parse holders json data.")
+        except (KeyError, IndexError) as e:
+            if not YfConfig().hide_exceptions:
+                raise
+            raise YFDataException(f"# RESPONSE:\n{result}\n\nFailed to parse 'holders' json response from Yahoo Finance (above): {str(e)}")
```

You probably want to raise a yf error either way, and is it supposed to raise `YfDataException` either way? Seems counter intuative